fiftyone/core/dataset.py

```diff
@@ -6809,7 +6809,10 @@ def _reload_docs(self, hard=False):
     def _serialize(self):
         return self._doc.to_dict(extended=True)
 
-    def _update_last_loaded_at(self):
+    def _update_last_loaded_at(self, force=False):
```

Comment:
There is probably many other cases where we do not need to do this like in API. But good start here.

Is there never a case where we want the server to update last loaded at though? I'm thinking of dataset selection from the dropdown, etc.
I see the `force` parameter but it does not seem to be used in this PR (passed from `serialize_dataset(update_last_loaded_at)` which is default `False` and not overridden)

Reply:
nvm, ctrl-f fail

Reply:
Yeah, the server should still update when loading a new dataset/view. Definitely more work/refinement to do here